graphql_compiler/schema.py

```diff
@@ -349,3 +349,15 @@ def insert_meta_fields_into_existing_schema(graphql_schema):
                                      .format(meta_field_name))
 
             type_obj.fields[meta_field_name] = meta_field
+
+
+def _check_for_nondefault_macro_directives(directives):
```

Comment:
The function can either leave a directive as-is, remove it from the schema(@output), or raise an error(any user-created directive). Since we don't want a DEFAULT_DIRECTIVES_DISALLOWED_IN_MACRO_EDGE_DEFINITION per [this comment](https://github.com/kensho-technologies/graphql-compiler/pull/313#commitcomment-33655951), we need whether all directives are default.